tensorflow_encrypted/config.py

```diff
@@ -83,6 +95,11 @@ def session(self, log_device_placement:bool=False) -> tf.Session:
                 intra_op_parallelism_threads=1
             )
         )
+        if __TFE_DEBUG__:
+            print('session in debug mode')
```

Comment:
debug print?

Reply:
?

Reply:
is it printing just for debugging purposes or should it be kept around?